test/fixtures/r-httr2/bash_redirect_heredoc.r

```diff
@@ -0,0 +1,12 @@
+library(httr2)
+
+headers = c(
+  `'Accept'` = "'application/json'",
```

Comment:
Too many quotes?

Reply:
This is basically from `curl http://example.com -H "'Accept': 'application/json'"`. So, it is parsed correctly I think. By the way this crashes `httr2::curl_translate()`

Reply:
Ah got it.